Finjector.Web/ClientApp/src/pages/Import.tsx

```diff
@@ -76,6 +77,24 @@ const Import = () => {
     );
   }
 
+  if (teamGroupsQuery.isLoading) {
```

Comment:
this should actually use 
```js
  const queryStatusComponent = useFinQueryStatusHandler({
    queryStatus: teamGroupsQuery,
  });
  if (queryStatusComponent) return <>{queryStatusComponent}</>;
```  
instead of `<FinLoader>` so that it handles errors too. but i like "we're testing your patience" lol